upcoming-release-notes/2153.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Enhancements
```

Comment:
```suggestion
category: Maintenance
```

@carkom as this isn't user-facing, I think this probably fits more with `Maintenance`